frontend/src/pages/registration/accountPage.js

```diff
@@ -18,11 +18,24 @@ function AccountPage() {
     const { user, logout } = useContext(AuthContext);
 
     const [isOpen, setIsOpen] = useState(false);
-
-    const handleDeleteAccount = () => {
+    const handleDeleteAccount = async () => {
         setIsOpen(false);
-        console.log("Акаунт видалено");  //future logic for delete
-        handleLogOut()
+    
+        if (!user?.email) {
+            toast.error("bajojajo bajojajo");
```

Comment:
I won't be looking any further. Please reexamine your PR and make it Ready For Review  :)